android/app/build.gradle

```diff
@@ -78,4 +78,4 @@ dependencies {
     androidTestImplementation 'androidx.test:runner:1.3.0-alpha03'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0-alpha03'
     implementation 'com.google.firebase:firebase-analytics:17.2.0'
-}
+}
```

Comment:
First of all sorry for being late to respond,
and talking about this file change I believe this is not needed over here.